setup.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Dag Wieers (@dagwieers) <dag@wieers.com>
```

Comment:
The copyright line seems out of place here.

Reply:
I will remove it, in other projects we use copyright statements in every file as is legally required.
In this case I would say *setup.py* is even not copyrightable.